src/components/Notifications/Notifications.actions.js

```diff
@@ -3,11 +3,12 @@ import {
   HIDE_NOTIFICATION
 } from './Notifications.constants';
 
-export function showNotification(message) {
+export function showNotification(message,kind = null) {
```

Comment:
Missing space after the comma. This should have been fixed automatically by Husky in a pre-commit hook, it might not be configured properly on your machine.